packages/apps/plugins/plugin-client/src/components/ClientSettings.tsx

```diff
@@ -18,6 +24,29 @@ export const ClientSettings = ({ settings }: { settings: ClientSettingsProps })
       <SettingsValue label={t('enable experimental automerge backend')}>
         <Input.Switch checked={settings.automerge} onCheckedChange={(checked) => (settings.automerge = !!checked)} />
       </SettingsValue>
+      <SettingsValue label={t('choose storage adaptor')}>
```

Comment:
is this a temporary setting along the lines of the automerge one?

Reply:
Probably yes